js/templates/nav-data.hbs

```diff
@@ -42,7 +42,7 @@
       </div>
     </div>
     <div class="mega-heading">
-      <h3 class="mega-heading__title icon-heading--table"><a href="{{webAppUrl}}/advanced">Advanced data</a></h3>
+      <h3 class="mega-heading__title icon-heading--table"><a href="{{webAppUrl}}/advanced">Advanced data - all</a></h3>
     </div>
```

Comment:
@noahmanger was this the correct file to edit? The changes weren't showing up locally, and I'm not sure familiar with how .hbs works, but it's the only place I could find the text.